Sources/Code/Record.swift

```diff
@@ -6,48 +6,48 @@
 //  Copyright © 2017 Jamit Labs GmbH. All rights reserved.
 //
 
-// NOTE: If you're using Realm, uncomment the code below and subclass `Record` instead of `Object` for your Realm objects. Also remove this files entry from .swiftlint.yml.
+//  NOTE: If you're using Realm, uncomment the code below and subclass `Record` instead of `Object` for your Realm objects. Also remove this files entry from .swiftlint.yml.
```

Comment:
Ah, I see now, you've unchecked the target. Please update the comment then to reflect your change.